resources/views/hardware/view.blade.php

```diff
@@ -281,21 +281,21 @@
                                                         <li> {{ $asset->model->manufacturer->name }}</li>
                                                     @endcan
 
-                                                    @if (($asset->model) && ($asset->model->manufacturer->url))
+                                                    @if (($asset->model->manufacturer) && ($asset->model->manufacturer->url!=''))
```

Comment:
We do actually have to keep the model check here, in case someone deleted a model directly from the database. In that case, `($asset->model->manufacturer) ` will 500, since we're looking for a property on a relationship that doesn't exist.